### ISSUE TITLE
HTML Boilerplate: Use note box for note

### ISSUE THREAD

**Issue by MaoShizhong** [TheOdinProject/curriculum#28373]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

![image](https://github.com/TheOdinProject/curriculum/assets/122839503/6405069b-ce64-40b2-884d-d05f01d2f22e)

We currently have an important note in the HTML Boilerplate lesson (shown above) that does not use one of our [note boxes](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#note-boxes). For better visibility, we should convert this to use a note box, specifically a warning note box.

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-html-boilerplate

**Comment by MaoShizhong** [comment]
## Acceptance criteria

Please comment below if you'd like to be assigned to work on this issue. Please do not open a PR unless you have been assigned by a maintainer.

- [ ] Convert the note shown above (under the `Viewing HTML files in the browser` heading) to use a note box.
- [ ] The note box must be of the warning type (`class="lesson-note lesson-note--warning"`).
- [ ] The note box must have a level 4 heading: `#### Use Google Chrome`

**Comment by mathdebate09** [comment]
Hey Mao, I can work this fix up :)
